nixos/modules/services/x11/window-managers/dwm.nix

```diff
@@ -0,0 +1,42 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+
+  cfg = config.services.xserver.windowManager.dwm;
+
+in
+
+{
+
+  ###### interface
+
+  options = {
+
+    services.xserver.windowManager.dwm.enable = mkOption {
```

Comment:
Quoting Arseniy Seroka (2015-12-03 08:27:59)

> > +    services.xserver.windowManager.dwm.enable = mkOption {
>
>    Please use mkEnableOption.

Can do. I based this off of another one of the files in that directory,
which still uses mkOption. I gather this is a deprecated way of doing
things that hasn't be cleaned up everywhere? Would a (separate) patch
that updates the rest be welcome?